nix/modules/scripts.nix

```diff
@@ -52,7 +52,7 @@
         # Want to update the root flake.lock if we're updating the version that is currently the default.
         if grep -qE "versions\.url = \".+\?dir=''${VERSIONS_DIR}\"" flake.nix; then
```

Comment:
before [a commit by @jost-s 12h ago](https://github.com/holochain/holochain/commit/3fe102926d52612fc06e94d41a06454691f6deaf) this was never actually hit, so the new codechange wouldn't have come into effect. now it will :+1: